src/astro/sql/operators/temp_hooks.py

```diff
@@ -56,3 +61,48 @@ def get_uri(self) -> str:
         if self.schema:
             uri += self.schema
         return uri
+
+
+class TempBigQueryHook(BigQueryHook):
+    def __init__(
+        self,
+        gcp_conn_id: str = GoogleBaseHook.default_conn_name,
+        delegate_to: Optional[str] = None,
+        use_legacy_sql: bool = True,
+        location: Optional[str] = None,
+        bigquery_conn_id: Optional[str] = None,
+        api_resource_configs: Optional[Dict] = None,
+        impersonation_chain: Optional[Union[str, Sequence[str]]] = None,
+        labels: Optional[Dict] = None,
+    ):
+        """Override __init__() method of BigQueryHook since it was not passing gcp_conn_id param to DbApiHook correctly."""
```

Comment:
Can you explain what you mean by "not correctly?" in the comment

Reply:
We no longer need this temp hook there has been a fix in upstream.